main.go

```diff
@@ -2,97 +2,119 @@ package main
 
 import (
 	"fmt"
-	log "github.com/sirupsen/logrus"
-	"net/http"
-	"os"
-
+	"github.com/sirupsen/logrus"
```

Comment:
why not `log "github.com/sirupsen/logrus"` ? it's ok for me...

Reply:
fixed